yw/Software/test/Checker1Test.java

```diff
@@ -0,0 +1,138 @@
+import static org.junit.Assert.*;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.StringTokenizer;
+
+import org.junit.Test;
+
+import spellchecker.Checker1;
+import spellchecker.Checker2;
+import spellchecker.Read;
+ 
+public class Checker1Test {
```

Comment:
I don't know why, but when I try to run this test class, I get a NoClassDefFoundError. Are your tests running on your local?